src/pages/index.html

```diff
@@ -4,16 +4,380 @@
+                    <span class="nomber2">+3.50%</span>
+                    <i class="fa-solid fa-arrow-up"></i>
+                  </div>
+     <div class=" d-flex flex-column">
+      <span class="nomber">86,590</span>
+      <span class="total">Total Customer</span>
+              </div>
+            </div>
+
+            <div class="squr2 position-absolute ">
+              <div class="d-flex">
+                <img src="../assets/images/aaaaa.svg" alt="">
+                <span class="nomber2">+48.50%</span>
+                <i class="fa-solid fa-arrow-up"></i>
+              </div>
+   <div class=" d-flex flex-column">
+    <span class="nomber">$36,845.00</span>
+    <span class="total">Total Revenue</span>
+  </div>
+   </div>
+
+
+        
+    <img src="../assets/images/person.png" alt="">
+  </div>
+
+</div>
+
+ </div>
+</div>
+
+
+
+</div>
+
+ <div class="page">
+  <div class="container">
+    <div class="developed">
+      <div class="updated">
+        <img src="../assets/images/icon (11).svg" alt="">
+        <p>  Developed with the highest care. Always updated. </p>
+      </div>
+      <div class="updated">
+        <img src="../assets/images/icon (12).svg" alt="">
+        <p> Get free goodies from all over the world after purchase </p>
+      
+      </div>
+      <div class="updated">
+        <img src="../assets/images/icon (13).svg" alt="">
+        <p>  Save up to 10% of every purchase you made </p>
+      
+      
+      </div>
+          </div>
+    <div class="data ">
+      <div class="lysis">
+      <div class="card1" >
+      <img src="../assets/images/Graph.png" alt="">
+          <div class="shap">
+<img src="../assets/images/shape (1).png" alt="">            
+                    </div>
+       
+        
+        </div>
+      
+        <div class="card2" >
+        
+          <h4> Communicative and fast data processing</h4>
+  <p> Importance of data processing includes better increased productivity
+     and profits, target, right decisions, more accurate and reliable.  </p>
+     <a href="#">  Make your first sale <i class="fa-solid fa-arrow-right"></i> </a>
+
+        </div>
+  
+      </div>
+      </div>
+
+   
+
+    <!-- <img src="../assets/images/Vector (1).svg" alt=""> -->
+
+
+
+
+</div>
+<div class="Payment">
+
+      
+  <div class="easiest  ">
+  
+  
+  
+  
+  <div class="card1" >
+  
+      <h3> Payment is the easiest with many options  </h3>
+      <p> Credit cards are easily the most secure and safe payment method to use when you shop online.  </p>
+  
+    <div class="Manage">
+      <div class="  d-flex  align-items-center">
+        <i class="fa-solid fa-check"></i>
+        <p>    Accept any method of payment</p>
+      </div>
+      <div class="  d-flex r align-items-center ">
+        <i class="fa-solid fa-check"></i>
+        <p>   Apply discounts and issue refunds</p>
+      </div>
+    </div>
+ <a href="#">  Make your first sale <i class="fa-solid fa-arrow-right"></i> </a>
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/6dcfad6b-a091-48fc-aed3-7d3cfcf97c58)
1- this is icons not same in design
2- space between p tag and anchor tag not same in design

Reply:
done